src/Illuminate/Foundation/helpers.php

```diff
@@ -994,7 +994,7 @@ function trans_choice($key, $number, array $replace = [], $locale = null)
     function __($key = null, $replace = [], $locale = null)
     {
         if (is_null($key)) {
-            return $key;
+            return null;
```

Comment:
It's exactly the same, because `$key` is already null!

Reply:
Yes! But why does it returns $key? This is not a bug fix, just making the code better and more readable.

Reply:
I think is already readable, but wait for taylor

Reply:
Thanks for your feedback.